packages/front-end/components/Settings/InviteModal.tsx

```diff
@@ -83,20 +98,30 @@ const InviteModal: FC<{ mutate: () => void; close: () => void }> = ({
       {emailSent === false && (
         <>
           <div className="alert alert-danger">
-            Failed to send invite email to <strong>{email}</strong>
-          </div>
-          <p>You can manually send them the following invite link:</p>
-          <div className="mb-3">
-            <code>{inviteUrl}</code>
+            {email.split(",").length > 1 ? (
+              "Failed to send the invite emails. To manually send the invite link, click the '3 dots' next to each invitee."
+            ) : (
+              <span>
+                Failed to send invite email to <strong>{email}</strong>
+              </span>
+            )}
           </div>
+          {email.split(",").length === 1 && (
+            <>
+              <p>You can manually send them the following invite link:</p>
+              <div className="mb-3">
+                <code>{inviteUrl}</code>
+              </div>
+            </>
+          )}
         </>
       )}
       {emailSent === null && (
         <>
           <Field
             label="Email Address"
-            type="email"
```

Comment:
We can keep `type="email"`.  We just need to add the `multiple` attribute as well to make it accept a comma-separated list.  That way we get to keep all of the built-in browser validation.